contrib/kubernetes/README.md

```diff
@@ -0,0 +1,44 @@
+# Kubernetes  Setup for DataHub
+
+## Introduction
+This directory provides the Kubernetes setup for DataHub.  This is the first version with simple YAML files.  
+The next version will contain DataHub [Helm](https://helm.sh/) chart that can be published to [Helm Hub](https://hub.helm.sh/)
+
+## Setup
+This kubernetes deployment doesn't contain the below artifacts. The idea is to use the original helm charts for deploying each of these separately.   
+
+* Kafka and Schema Registry [Chart Link](https://hub.helm.sh/charts/incubator/kafka)
```

Comment:
We depend on Confluent Kafka right now, changing dependency to Apache Kafka might need small refactoring in the codebase I suppose (expect minimal change on kafka producer & consumer). But, this is not a big deal, we can change it to Apache Kafka. The only reason we depended on Confluent Kafka was ease of use because they had their Docker images ready in Docker Hub. It was so easy to use.

Reply:
Confluent Helm charts are also available.  Need to test it e2e.  Will change the link to point to this https://github.com/confluentinc/cp-helm-charts/tree/master/charts/cp-kafka

Reply:
Sure, that'd be easier.

Reply:
Done! Changed the Chart Link